### PR TITLE
Add FXIOS-7205 [v116] More logs again investigation migration

### PR DESCRIPTION
**Pull request by lmarceau**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-7205)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/15989)

## :bulb: Description
More logs again investigation migration

## :pencil: Checklist
You have to check all boxes before merging
- [X] Filled in the above information (tickets numbers and description of your work)
- [X] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [X] Wrote unit tests and/or ensured the tests suite is passing
- [X] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [X] If needed I updated documentation / comments for complex code and public methods

### CODE REVIEW

**Review by nbhasin2 — APPROVED** [review]

**Review by OrlaM — APPROVED** [review]

**Comment by lmarceau** [comment]
@Mergifyio backport release/v116.2

File: Client/Application/AppLaunchUtil.swift [hunk 1]
```diff
@@ -84,6 +84,10 @@ class AppLaunchUtil {
         logger.log("App version \(AppInfo.appVersion), Build number \(AppInfo.buildNumber)",
                    level: .debug,
                    category: .setup)
+
```
Comment (ahmednaher) [inline Client/Application/AppLaunchUtil.swift:87]: 2/1

Comment (ahmednaher) [reply Client/Application/AppLaunchUtil.swift:87]: 1/2

**Comment by rvandermeulen** [comment]
@mergifyio backport release/v117

**Pull request merged**